opencti-platform/opencti-graphql/src/migrations/1687529127415-triggers-remove-unused-field.js

```diff
@@ -0,0 +1,55 @@
+import { elUpdateByQueryForMigration } from '../database/engine';
+import { DatabaseError } from '../config/errors';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+
+const updateTriggers = async () => {
+  const updateQuery = {
+    script: {
+      params: { toRemoveFields: ['user_ids', 'group_ids', 'recipients'] },
+      source: 'for(def field : params.toRemoveFields) ctx._source.remove(field)'
+    },
+    query: {
+      bool: {
+        must: [
+          {
+            term: {
+              'entity_type.keyword': {
+                value: 'trigger'
+              }
+            }
+          }
+        ],
+        should: [
+          {
+            exists: {
+              field: 'user_ids'
+            }
+          },
+          {
+            exists: {
+              field: 'group_ids'
+            }
+          },
+          {
+            exists: {
+              field: 'recipients'
+            }
+          }
+        ],
+        minimum_should_match: 1
+      }
+    }
+  };
+  const message = '[MIGRATION] Triggers remove unused fields: recipients, user_ids, group_ids';
+  return elUpdateByQueryForMigration(message, READ_INDEX_INTERNAL_OBJECTS, updateQuery).catch((err) => {
+    throw DatabaseError('Error updating elastic', { error: err });
+  });
+};
+export const up = async (next) => {
+  await updateTriggers();
```

Comment:
Why do we have a method instead of code ?

Reply:
Not sure to understand your comment: the method called is defined above the `up`